pkgs/top-level/all-packages.nix

```diff
@@ -31,6 +31,11 @@ with pkgs;
     then pkgsi686Linux.callPackage
     else throw "callPackage_i686 not supported on system '${stdenv.system}'";
 
+  # Useful attribute of systems to cross compile to.
+  pkgsCross = lib.mapAttrs (n: crossSystem:
+              nixpkgsFun { inherit crossSystem; })
+              lib.systems.examples;
```

Comment:
Do people like doing this? We do it in reflex-platform & it works out nicely for convenience.

/cc @Ericson2314

Reply:
The main thing is that packages inside nixpkgs shouldn't depend on this. Maybe we can make it some sort of default overlay that the tarball job can disable?

Reply:
Sounds good